packages/react-native-gesture-handler/apple/RNGestureHandlerDetector.mm

```diff
@@ -94,11 +94,13 @@ - (BOOL)shouldAttachGestureToSubview:(NSNumber *)handlerTag
   return [[[handlerManager registry] handlerWithTag:handlerTag] wantsToAttachDirectlyToView];
 }
 
-- (void)addSubview:(UIView *)view
+- (void)didAddSubview:(UIView *)view
 {
-  [super addSubview:view];
+  for (const id handlerTag : _attachedNativeHandlers) {
+    [self tryAttachHandlerToChildView:handlerTag];
+  }
```

Comment:
Why was this loop moved outside of `tryAttachHandlerToChildView`? 

Also we could find better name for this method, like `tryAttachNativeHandlersToChildView`